gr-analog/python/analog/wfm_rcv_pll.py

```diff
@@ -39,14 +39,13 @@ def __init__(self, demod_rate, audio_decimation):
         ##################################################
         # Variables
         ##################################################
-        self.samp_rate = samp_rate = 3840000
-        self.rf_decim = rf_decim = 10
-        self.demod_rate = demod_rate = (int)(samp_rate/rf_decim)
-        self.stereo_carrier_filter_coeffs_0 = stereo_carrier_filter_coeffs_0 = firdes.band_pass(1.0, demod_rate, 37600, 38400, 400, firdes.WIN_HAMMING, 6.76)
+        self.demod_rate = demod_rate
+        self.stereo_carrier_filter_coeffs_0 = stereo_carrier_filter_coeffs_0 = firdes.band_pass(-2.0, demod_rate, 37600, 38400, 400, firdes.WIN_HAMMING, 6.76)
         self.stereo_carrier_filter_coeffs = stereo_carrier_filter_coeffs = firdes.complex_band_pass(1.0, demod_rate, 18980, 19020, 1500, firdes.WIN_HAMMING, 6.76)
         self.deviation = deviation = 75000
         self.audio_filter = audio_filter = firdes.low_pass(1, demod_rate, 15000,1500, firdes.WIN_HAMMING, 6.76)
-        self.audio_decim = audio_decim = (int)(demod_rate/48000)
+        self.audio_decim = audio_decim = audio_decimation
+        self.samp_delay = samp_delay = (int)((len(stereo_carrier_filter_coeffs)-1)/2 + (len(stereo_carrier_filter_coeffs_0)-1)/2)
```

Comment:
Python doesn't have c-style cast. Just use the // operator (floor division):
`(len(stereo_carrier_filter_coeffs) - 1) // 2 + (len(stereo_carrier_filter_coeffs_0) - 1) // 2`

Reply:
Thanks @dave-pi, I've updated and pushed changes. Also squashed multiple commits into one.